app/models/user.rb

```diff
@@ -0,0 +1,3 @@
+class User < ApplicationRecord
+  validates :username, :email, :password, presence: true
```

Comment:
- [ ] [OPTIONAL] How about validating the size and pattern for the email and password too? What about the username, does it have to be unique? How could you validate that?

Reply:
The following validations have now been added to the User model:

1. Both username and e-mail now have a **uniqueness** validator.
2. E-mail must be in the **format** of a regex pattern for an e-mail address.
3. The password has a **length** validator, requiring at least 6 characters.